src/vk_mem_alloc.h

```diff
@@ -3329,7 +3329,7 @@ VMA_CALL_PRE VkResult VMA_CALL_POST vmaResizeAllocation(
 
 /** \brief Returns current information about specified allocation and atomically marks it as used in current frame.
 
-Current paramters of given allocation are returned in `pAllocationInfo`.
+Current paramteres of given allocation are returned in `pAllocationInfo`.

```

Comment:
This is also wrong, it should be "parameters"

Reply:
The irony. :) Sorry I didn't get to fix this in time. :/